templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,237 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-hello-simple">
+    <style>
+      #{{ snippet_id }}-ffos-circling-fox {
+          /* All CSS should be preceeded by the ID selector for this individual
+            snippet. */
+      }
+    </style>
+    
+    {% if icon_url %}<a href="#" id="hello-cta">{% endif %}
```

Comment:
Also due to the way you've hooked up the events for this, only the icon will open the hello dialog. If you use a class then an admin can add a link in the snippet text that also opens up the hello dialog: `<a href="#" class="launch-hello">Click here to get started!</a>`.